battleship-client/src/main/java/com/academy/solid/nie/client/ui/Board.java

```diff
@@ -32,9 +30,8 @@ BoardFX getBoardFX() {
         return boardFX;
     }
 
-    Board(final boolean enemy, final Output output) {
+    Board(final boolean enemy) {
```

Comment:
boolean final in this case looks a little strange

Reply:
ok